lib/pbench/server/api/resources/__init__.py

```diff
@@ -470,6 +490,7 @@ class ParamType(Enum):
     KEYWORD = ("Keyword", convert_keyword)
     LIST = ("List", convert_list)
     STRING = ("String", convert_string)
+    INT = ("Int", convert_int)
```

Comment:
Fine; I'd prefer `INTEGER = ("Integer", convert_integer)` since none of the others are abbreviated. (Though JSON is an acronym...) But as you've resisted this for several iterations I guess I'll give up.